manifests/kubernetes/kubevirt-tekton-tasks-kubernetes.yaml

```diff
@@ -8,6 +8,13 @@ metadata:
     tekton.dev/tags: kubevirt
     tekton.dev/displayName: "KubeVirt cleanup VM"
     tekton.dev/platforms: "linux/amd64"
+    artifacthub.io/maintainers: |
+      - name: Karel Simon
+        email: ksimon@redhat.com
+    artifacthub.io/provider: KubeVirt
+    artifacthub.io/recommendations: |
```

Comment:
Makes sense to put the repo URL here too?

Reply:
Hi, there is already repo link in the ArtifactHub repository, e.g. [1] you can see source link.

[1] https://artifacthub.io/packages/tekton-task/kubevirt-disk-uploader/kubevirt-disk-uploader-task